src/devices/bus/nes_ctrl/zapper.cpp

```diff
@@ -64,6 +99,12 @@ void nes_zapper_device::device_start()
 {
 }
 
+void nes_bandaihs_device::device_start()
+{
+	save_item(NAME(m_latch));
+	save_item(NAME(m_strobe));
+}
```

Comment:
You should really call `nes_zapper_device::device_start()` even if it does nothing.  It’s good practice to start the base device class.

Reply:
Done.